neckmk3_2joints experimental setup (#689)
robotology/robots-configuration: experimentalSetups/neckmk3_2joints/README.md

2 Joints Neck Setup

> ===================

Reviewer: ```suggestion
===================

```

Leave a blank line.

This setup is a prototype of the neck that will be mounted on `iCub` and `ergoCub`.          

> It comprehends two motors and it can be moved using `yarprobotinterface` and `yarpmotorgui`.   

Reviewer: ```suggestion
It consists of two motors and can be actuated using `yarprobotinterface` and `yarpmotorgui`.   
```

One motor is controlled by an `amc` board and the other by an `amcbldc` board.
All the parameters to move the setup are in the configuration files.   

> ### Photo of the setup 

Reviewer: ```suggestion
### Picture of the setup 
```
